healthScore/models.py

```diff
@@ -43,43 +43,53 @@ class hospitalStaff(models.Model):  # Viewed by hospitalAdmin
 
 
 class CustomUserManager(BaseUserManager):
-    def create_user(self, email, username, password=None, **extra_fields):
+    def _create_user(self, email, password, **extra_fields):
         if not email:
-            raise ValueError("Users must have an email address")
-        user = self.model(
-            email=self.normalize_email(email), username=username, **extra_fields
-        )
+            raise ValueError("You have not provided a valid e-mail address")
```

Comment:
I think this email validation would be happening in the UI itself. Could you please check and update?

Reply:
Hi, this is not only a validation for UI, but also for CLI and related Python code. This part referred to the official Django documentation https://docs.djangoproject.com/en/5.0/topics/auth/customizing/#a-full-example

Reply:
Ahh, alright! Approving the code